### PR TITLE
Added index.md C

### PR DESCRIPTION
**Pull request by Tsikitsiky**
Hey Team,
This is our work, please have a look and comment.  Happy to change if needs to.

Thanks,
Sarah and Sandy

### CODE REVIEW

File: c/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# List of the artists starting with C
+
+- [Claudiot](https://claudiot.md)
```
Comment (ndorfin) [inline c/index.md]: This looks good, you won't need the `https://` scheme in the URL though, just `claudiot.md`

File: c/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- [Claud de Mahanoro](https://claud-de-mahanoro.md)
+- [Clo mahajanga](http://clo-mahajanga)
+- [Clauvis](http://clauvis.md)
+- [Chantal](http://chantal.md)
+
+[Go back to the artist index](http://../index.md)
```
Comment (ndorfin) [inline c/index.md]: Same with this

File: c/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# List of the artists starting with C
```
Comment (ndorfin) [inline c/index.md]: What is the title meant to say?

**Review by ndorfin — CHANGES_REQUESTED** [review]
I'd suggest changing the title of your PR to reflect what letter you were working on.
Once you make all these changes, please let me know, so I can re-view it.

File: c/index.md (hunk 3 above)
Comment (Tsikitsiky) [reply c/index.md]: We don't understand,is it the list title or the code title please?

Comment (Tsikitsiky) [reply c/index.md]: Shaun,could please check again, we've made changes.

**Review by ndorfin — CHANGES_REQUESTED** [review]
Just the title needs to change.
It's meant to be: `List of artists starting with C`

**Comment by ndorfin** [comment]
Everything else looks perfect!

File: c/index.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# List of artists starting with C
+
+- [Claudiot](claudiot.md)
+- [Claud de Mahanoro](claud-de-mahanoro.md)
+- [Clo mahajanga](clo-mahajanga.md)
+- [Clauvis](clauvis.md)
+- [Chantal](chantal.md)
```
Comment (ndorfin) [inline c/index.md]: Please arrange this list alphabetically.

**Review by ndorfin — CHANGES_REQUESTED** [review]

File: c/index.md (hunk 4 above)
Comment (Tsikitsiky) [reply c/index.md]: Hey, 
Here it is, please check.

File: c/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# List of artists starting with C
+
+- [Chantal](chantal.md)
+- [Claud de Mahanoro](claud-de-mahanoro.md)
+
```
Comment (ndorfin) [inline c/index.md]: Please remove these extra empty lines

File: c/index.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+- [Claudiot](claudiot.md)
+
```
Comment (ndorfin) [inline c/index.md]: Please remove these extra empty lines

File: c/index.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+- [Clauvis](clauvis.md)
+- [Clo mahajanga](clo-mahajanga.md)
```
Comment (ndorfin) [inline c/index.md]: Is `mahajanga` meant to have a capital letter?

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Review by ndorfin — APPROVED** [review]
LGTM! 🚢 -it!

**Pull request merged**